thoth/lab/inspection.py

```diff
@@ -457,13 +478,246 @@ def show_categories(inspection_df: pd.DataFrame):
     """List categories in the given DataFrame."""
     index = inspection_df.index.droplevel(-1).unique()
 
+    results_categories = {}
     for n, idx in enumerate(index.values):
-        print("\nCategory {}/{}".format(n + 1, len(index)))
+        print("\nClass {}/{}".format(n + 1, len(index)))
+
+        class_results = {}
         if len(index.names) > 1:
             for name, ind in zip(index.names, idx):
                 print(f"{name} :", ind)
+                class_results[name] = ind
         else:
             print(f"{index.names[0]} :", idx)
+            class_results[index.names[0]] = idx
+        results_categories[n + 1] = class_results
 
         frame = inspection_df.loc[idx]
         print("Number of rows (jobs) is:", frame.shape[0])
+
+    return results_categories
+
+
+def create_inspection_results_batches_dict(inspection_results: List[dict], inspection_results_list: List[str]) -> dict:
```

Comment:
Isn't the `generate_inspection_resuts` function (defined in https://github.com/thoth-station/notebooks/pull/46) missing from this PR?